test/support/codeflare.go

```diff
@@ -50,6 +61,30 @@ func GetPyTorchImage() string {
 	return lookupEnvOrDefault(CodeFlareTestPyTorchImage, "pytorch/pytorch:1.11.0-cuda11.3-cudnn8-runtime")
 }
 
+func GetInstaScaleOcmSecretName() string {
+	return lookupEnvOrDefault(InstaScaleOcmSecretName, "instascale-ocm-secret")
+}
+
+func GetInstaScaleOcmSecretNamespace() string {
+	return lookupEnvOrDefault(InstaScaleOcmSecretNamespace, "default")
+}
```

Comment:
Can this be collapsed into a single environment variable, so it's referenced as `default/instascale-ocm-secret` for example.

Reply:
Maybe but would it make sense when the variables are used separately in the Secret(namespace) and Get(secret name) parts here: 
```
instascaleOCMSecret, err := test.Client().Core().CoreV1().Secrets(GetInstaScaleOcmSecretNamespace()).Get(test.Ctx(), GetInstaScaleOcmSecretName(), metav1.GetOptions{})
```

Reply:
I'd rather have a single env variable for clients to set, e.g. `INSTASCALE_OCM_SECRET=default/instacale-ocm-secret` and the two functions adapted so they split the provided string, or a single function that returns both strings, e.g.:

```go
func GetInstaScaleOcmSecret() (string, string) {
	res := strings.SplitN(lookupEnvOrDefault(InstaScaleOcmSecret, "default/instascale-ocm-secret"), "/", 2)
	return res[0], res[1]
}
```

Reply:
Ok, thank you. I have updated.